website/content/docs/ecs/resource-usage.mdx

```diff
@@ -0,0 +1,33 @@
+---
+layout: docs
+page_title: Resource Usage - AWS ECS
+description: >-
+  Resource usage of the Consul Service Mesh on AWS ECS (Elastic Container Service).
+---
+
+# Resource Usage
+
+We ran performance tests on Consul on ECS to determine the resource usage of
+the sidecar containers Consul on ECS injects along with the ACL controller.
+[The architecture page](/docs/ecs/architecture) describes each Consul on ECS
+component in depth.
+
```

Comment:
Would the paragraph starting on line15 be easier to consume in list format, i.e.:

We used the following procedure to measure resource usage:
* Executed performance tests while deploying clusers of various sizes. We ensured that deployment conditions stressed Consul on ESC components.
* After each performance test session, we recorded resource usage for each component to determine worst-case scenario resource usage in a production environment. 
* We used Fargate's minimum allowed CPU (256 shares) and memory settings (512 MB) on ECS during performance testing to demonstrate that Consul on ECS along with application containers can run on the smallest ECS tasks.